api/filters/replacement/replacement.go

```diff
@@ -70,7 +70,18 @@ func selectSourceNode(nodes []*yaml.RNode, selector *types.SourceSelector) (*yam
 			return nil, fmt.Errorf("error getting node IDs: %w", err)
 		}
 		for _, id := range ids {
-			if id.IsSelectedBy(selector.ResId) {
+			if id.IsSelectedBy(selector.ResId) && !rejectId(selector.Reject, &id) {
```

Comment:
It seems like the tests are failing because rejectId is undefined

from the pre submit master tests output
```
undefined: rejectId
```

Reply:
@morix1500 can you check into this ?